internal/datanode/data_node.go

```diff
@@ -993,7 +993,7 @@ func (node *DataNode) Import(ctx context.Context, req *datapb.ImportTaskRequest)
 	defer cancel()
 	// func to report import state to RootCoord.
 	reportFunc := func(res *rootcoordpb.ImportResult) error {
-		status, err := node.rootCoord.ReportImport(newCtx, res)
+		status, err := node.rootCoord.ReportImport(ctx, res)
```

Comment:
@soothing-rain specified this context cannot use `ctx` for reporting since if ctx maybe canceled and failure report may never reach rootcoord.
Any change on this?

Reply:
confirmed offline with @yhmo @soothing-rain 
this shall be refined to async call in next PRs